tests/test_database_client.py

```diff
@@ -212,14 +212,14 @@ def test_update_last_cached(live_database_client):
     # Add a new data source to the database
     insert_test_agencies_and_sources_if_not_exist(live_database_client.cursor)
     # Update the data source's last_cached value with the DatabaseClient method
-    new_last_cached = datetime.now()
+    new_last_cached = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
Yeah, it does make more sense to have the `strftime` method alongside everything else, rather than down at the bottom. Good catch 🥅

Reply:
I believe I did it this way since `datetime` in Postgres will accept milliseconds and I didn't want to deal with all that in there!